.pre-commit-config.yaml

```diff
@@ -106,3 +106,8 @@ repos:
         entry: ./activated.py flake8
         language: system
         types: [python]
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.7.1
+    hooks:
+      - id: ruff
+        args: ["--fix"]
```

Comment:
I'm at least tempted to run it out of our local venv like other tools here.  The primary upside is a single definition of the version we use in a form that all our existing dependencies tools are monitoring.

Reply:
Good idea, I didn't think about that.